src/main/java/winstone/HostConfiguration.java

```diff
@@ -117,7 +114,9 @@ private void loadBuiltinMimeTypes() {
             Properties props = new Properties();
             props.load(in);
             for (Entry<Object, Object> e : props.entrySet()) {
-                mimeTypes.addMimeMapping(e.getKey().toString(), e.getValue().toString());
+                this.server
```

Comment:
A few lines above on line 97, you don't qualify the access to the server field (`server.getMimeTypes().addMimeMapping(extension.toLowerCase(), mimeType);`), so for consistency I don't think we should qualify the access here either.

Reply:
line 103 and 97 are part of the constructor. so it's a different context.
anyway, this is really a nit comment. If you want to change this once the PR is merged. I will be happy to approve the PR but now you reached my limit of handling nit comments. 
It's time to move forward and get this done...